### PR TITLE
Dark-mode implemented

### PR DESCRIPTION
**Pull request by raashika03**
closed #71 cause that had changes for other issues also, 
This's a clean PR with changes only related to this issue #60

### CODE REVIEW

**Comment by raashika03** [comment]
Hi @bryan-brancotte can this be merged if This's all OK. 
Further on I've thought to do something like this 
IMG_20210411_060910
Will put that theme button into some Menu with some other options as discussed in #62 #77, It'll be better from designing point of view. 
But since that's quite different issue from this, so want to handle this thing in some other pr. 
Then it'll be easier for me to work on.

**Comment by raashika03** [comment]
Or you can check this source (https://www.covid19india.org/) menu option, where that theme functionality is added. 

Will take inspiration for the basic design from above website and for positioning will take from gitter shown above.

**Comment by raashika03** [comment]
No sorry this can't be done as now only, I noticed that splashscreen/about menu discussed to be implemented in #77 will be positioned in the footer so going on with the theme button at the present position seems good. It has to be at the top part of the main page. So this's all ok. No work needed further.

**Comment by raashika03** [comment]
@bryan-brancotte any comments

**Review by hmenager — CHANGES_REQUESTED** [review]
Hi @raashika03 it looks like I forgot to finalize my review and that means you couldn't see my comments. I am very very sorry for that! I was wondering myself why you weren't answering :flushed:. Can you have a look at my comments and see what you can or cannot do?

File: edit.html [hunk 1]
```diff
@@ -23,13 +23,34 @@
...
 </head>
 <body>
 <div class="container">
     <div class="page-header row">
+        <button class="btn-toggle" title="Change theme mode">
+<svg style="fill: var(--color-profile-color-modes-toggle-moon);" aria-hidden="true" width="20" height="20" viewBox="0 0 14 13" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M4.52208 7.71754C7.5782 7.71754 10.0557 5.24006 10.0557 2.18394C10.0557 1.93498 10.0392 1.68986 10.0074 1.44961C9.95801 1.07727 10.3495 0.771159 10.6474 0.99992C12.1153 2.12716 13.0615 3.89999 13.0615 5.89383C13.0615 9.29958 10.3006 12.0605 6.89485 12.0605C3.95334 12.0605 1.49286 10.001 0.876728 7.24527C0.794841 6.87902 1.23668 6.65289 1.55321 6.85451C2.41106 7.40095 3.4296 7.71754 4.52208 7.71754Z"></path>
+          </svg>
+        </button>
+        <script>const btn = document.querySelector(".btn-toggle");
```
[the remark this answers is not in the document]
Comment (raashika03) [reply edit.html:39]: Ok let me try to put it in utils.js.
At first I tried to put it in some js file (index.js) only but then it didn't work, couldn't figure out about utils.js though. So finally left it in the HTML.
But now I'll try again. Thank you @hmenager for the review.

File: css/index.css [hunk 2]
```diff
@@ -125,3 +125,182 @@ body {
 .branch {
     cursor: pointer;
 }
+
```
[the remark this answers is not in the document]
Comment (raashika03) [reply css/index.css:128]: Ok you mean I should create a separate file named as Dark-mode.css kind, then will import it in both index.html and edit.html. 
At present I put the css for all the elements of front page in index.css then for specific elements present only in propose change pages for which overriding is needed, I kept in bootstrap.xl.css. And imported index.css in edit.html for using all the basic same styling.

Or do you mean I should keep all in index.css?

File: index.html [hunk 3]
```diff
@@ -134,6 +134,26 @@ <h4>EDAM is a simple ontology of well established, familiar concepts that are pr
         </div>
     </div>
     <div class="content row" id="main">
+        <button class="btn-toggle" title="Change theme mode">
```
[the remark this answers is not in the document]
Comment (raashika03) [reply index.html:137]: Yes, I was thinking the same. But was waiting for mentors to reply for the positioning of the button.
Yes, it disaligns that panel. let me see where it'll fit nicely.

File: css/index.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (raashika03) [reply css/index.css:128]: @hmenager Please reply here asap. Needs a small clarity.

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (raashika03) [reply index.html:137]: But bottom-right do you mean in the line of previously seen option or exact at the bottom right in the line of communication logos. But positioning theme button at the bottom isn't a nice idea. It's not easy to notice if it's at the bottom of the page.

[the remark this answers is not in the document]
Comment (raashika03) [reply index.html:137]: Or I can place it at the position of "collapse not selected elements" where it's empty now after positioning that on the tree map. so bottom left sounds better than bottom right.

**Comment by raashika03** [comment]
Showing the theme button position for both desktop/ smartphone sizes. I placed it at the bottom left of tree-map cause there's no space on the bottom-right side.
Screenshot from 2021-04-22 22-56-24
Screenshot from 2021-04-22 22-56-03

**Comment by raashika03** [comment]
I removed this [error](https://github.com/edamontology/edam-browser/pull/96/commits/2f39ca8394eae8f15ad5023259001e1a0b634170) which got into this commit.
So, everything is ok now.

**Review by hmenager — CHANGES_REQUESTED** [review]
Hi @raashika03 thanks a lot for all these improvements (and I'm very sorry for not finalizing my previous review earlier). Do you think you can make it so that the function defined in both index.js and edit.js is only defined once in utils.js (or another standalone js file if you prefer)? Please let me know if help is needed.

**Comment by hmenager** [comment]
> I removed this [error](https://github.com/edamontology/edam-browser/pull/96/commits/2f39ca8394eae8f15ad5023259001e1a0b634170) which got into this commit.
> So, everything is ok now.
> 
> I tried to remove that single line change in bootstrap.xl.css.
> But couldn't don't know why sweat

No worries about this, this is probably only a minor carriage return added automatically by your editor. :+1:

**Comment by raashika03** [comment]
Done @hmenager :sweat_smile: I should've done that before but I missed it anyhow while changing so many things.

**Review by hmenager — APPROVED** [review]
Looks good to me, thank you @raashika03 !

**Pull request merged**